ansible/roles/kubernetes-common/templates/etc/kubernetes/apiserver-audit-policy.yaml

```diff
@@ -0,0 +1,162 @@
+apiVersion: audit.k8s.io/v1beta1
+kind: Policy
+rules:
+{% if kubernetes_common_audit_policy_rules is defined and kubernetes_common_audit_policy_rules != "" %}
```

Comment:
Do we define a default value for this?  I think this conditional should be checking the length of the `kubernetes_common_audit_policy_rules` list.

This pattern is fine, but it is a bit different than how we have done other API templates. In those templates we have put the full spec into the defaults so that users can override values using ansible's merge hash functionality. As this is primarily a list of rules, this would be much harder to override specific parts.

Reply:
Ah yep! Will change to follow the existing pattern. Thanks 👍